### PR TITLE
Remove title

### PR DESCRIPTION
**Pull request by Yegor-Tarshin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# Go Rails
```
Comment (Kesha1407) [inline README.md:1]: (_*_)

Comment (Kesha1407) [inline README.md:1]: (_*_)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
+
```
Comment (alexjavascript) [inline README.md]: Why do you add this empty line? Maybe you could remove it if needed

File: README.md (hunk 1 above)
Comment (Kesha1407) [inline README.md:1]: Wazzup

File: README.md (hunk 2 above)
Comment (Yegor-Tarshin) [reply README.md]: Thank you, Master!

Comment (BabyDiv) [reply README.md]: *if it's needed
It's passive voice in Present SImple

**Review by alexjavascript — APPROVED** [review]

**Pull request merged**